pytest_splunk_addon/standard_lib/app_test_generator.py

```diff
@@ -120,11 +120,8 @@ def generate_tests(self, fixture):
                         test_type="line_breaker"
                         )
                 )
-            if isinstance(pytest_params, str):
-                LOGGER.warning(pytest_params)
 
-            elif pytest_params:
-                yield from sorted(pytest_params, key=lambda param: param.id)
+            yield from sorted(pytest_params, key=lambda param: param.id)
```

Comment:
I decided to remove unreachable code, let me know if you see the need for additional exception handling